utils/spacewalk-common-channels.ini

```diff
@@ -955,14 +955,23 @@ base_channels = ubuntu-16.04-pool-amd64
 repo_url = http://archive.ubuntu.com/ubuntu/dists/xenial-security/main/binary-amd64/
 
 [ubuntu-1604-amd64-universe]
-label    = ubuntu-1604-amd64-main-universe
```

Comment:
It seems this was a typo, but please check the doc, because I think it needs an update there as well.

Reply:
Doc is correct.